watertap/unit_models/electrodialysis_1D.py

```diff
@@ -580,12 +586,28 @@ def build(self):
             units=pyunits.meter * pyunits.second**-1 * pyunits.pascal**-1,
             doc="Water permeability coefficient",
         )
-        self.membrane_areal_resistance = Var(
+        self.membrane_areal_resistance_x = Var(
+            self.membrane_set,
+            self.diluate.length_domain,
+            initialize=2e-4,
+            bounds=(1e-6, 1),
+            units=pyunits.ohm * pyunits.meter**2,
+            doc="Areal resistance of membrane",
+        )
+        self.membrane_areal_resistance_const = Var(
             self.membrane_set,
             initialize=2e-4,
             bounds=(1e-6, 1),
             units=pyunits.ohm * pyunits.meter**2,
-            doc="Surface resistance of membrane",
+            doc="Constant areal resistance of membrane at high electrolyte concentration",
```

Comment:
So, the membrane areal resistance is dominated by the diluate but the constant is set by the concentrate?

Reply:
yes. I hesitated about using the infinite word as it's not practically meaningful.  but I'll make an update.

Reply:
this is updated to "infinity-approximated electrolyte concentration"